src/dodal/devices/electron_analyser/vgscienta/vgscienta_analyser.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+
+from ophyd_async.core import AsyncStatus
+from ophyd_async.epics.core import epics_signal_rw
+
+from dodal.devices.electron_analyser.base_analyser import BaseAnalyser
+from dodal.devices.electron_analyser.base_region import EnergyMode
+from dodal.devices.electron_analyser.vgscienta.vgscienta_region import (
+    DetectorMode,
+    VGScientaRegion,
+)
+
+
+class VGScientaAnalyser(BaseAnalyser):
```

Comment:
Should: As in base class re constants, `_signal`, `self.prefix` and use of `set`